src/modules/xplorers-ai/xplorersAISearch.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import React, { ChangeEvent, MouseEvent } from "react";
+import Image from "next/image";
+import attachmentIcon from "@public/icons/attachment.svg";
+import searchIcon from "@public/icons/search.svg";
+import { FileUploadEvent } from "@/hooks/useXplorersAI";
+interface SearchProps {
+  handleInputChange: (e: ChangeEvent<HTMLTextAreaElement>) => void;
+  handleSubmit: (e: MouseEvent<HTMLButtonElement>) => Promise<void>;
+  handleFileUpload: (e: FileUploadEvent) => void;
+  value: string;
+}
+const XplorersAISearch: React.FC<SearchProps> = ({
```

Comment:
Needs Proper gap between imports, interface and start of component.

Reply:
Also, prop drilling is happening here, why not create a global context for handling search ?